### PR TITLE
📝 Add a technical blog post to explain how to run anomalib.

### PR DESCRIPTION
**Pull request by ashwinvaidya17**
# Description

- Add blog to development

### CODE REVIEW

File: docs/blog/biz/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+ # Anomalib
```
Comment (samet-akcay) [inline docs/blog/biz/README.md:1]: I think `docs/blog/biz/README.md` is not ready to be merged. Instead, we should only merge the technical blog post.

File: docs/blog/tec/README.md [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+
+Anomalib aims to collect the most recent deep-learning based anomaly detection algorithms with the purpose of providing a tool that makes it easy to benchmark different anomaly detection algorithms on public and custom datasets. Anomalib is continuously updated with the latest State-of-the-Art  algorithms, and contains several tools and interfaces that can be used to run experiments and create and test new algorithms.
+
+## How to train a Custom dataset with Anomalib
+
+Anomalib supports a number of datasets in various formats, including the state-of-the-art anomaly detection benchmarks such as MVTec AD and BeanTech. For those who would like to use the library on their custom datasets, anomalib also provides a `FolderDatasetModule` that can load datasets from a folder on a file system. The scope of this post will be to train anomalib models on custom datasets using the `FolderDatasetModule`.
```
Comment (samet-akcay) [inline docs/blog/tec/README.md:18]: `FolderDatasetModule` has recently been refactored to `Folder`

File: docs/blog/tec/README.md [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+### Step 2: Collect Your Custom Dataset
+Anomalib supports multiple image extensions such as `".jpg", ".jpeg", ".png", ".ppm", ".bmp", ".pgm", ".tif", ".tiff", and ".webp"`. A dataset can be collected from images that have any of these extensions.
+
+### Step 3: Format your dataset
+Depending on the use-case and collection, custom datasets can have different formats, some of which are listed below:
+-   A dataset with good and bad images.
+-   A dataset with good and bad images as well as mask ground-truths for pixel-wise evaluation.
+-   A dataset with good and bad images that is already split into training and testing sets.
+
+Each of these use-cases is addressed by anomalib's  `FolderDataModule`. Let's focus on the first use-case as an example of end-to-end model training and inference. In this post, we will use a toy dataset which you can download from [here](https://openvinotoolkit.github.io/anomalib/_downloads/3f2af1d7748194b18c2177a34c03a2c4/hazelnut_toy.zip). The dataset consists of several folders, each containing a set of images. The _colour_ and the _crack_ folders represent two kinds of defects. We can ignore the _masks_ folder for now.
```
Comment (samet-akcay) [inline docs/blog/tec/README.md:45]: Same as above

**Review by samet-akcay — CHANGES_REQUESTED** [review]
Thanks, getting there. Only few comments.

**Review by samet-akcay — APPROVED** [review]

File: docs/blog/001-train-custom-dataset/README.md [hunk 4]
```diff
@@ -0,0 +1,245 @@
+# How to Train Your Custom Dataset with Anomalib
+<div align="center">
+<img src="docs/blog/../../images/anomalib-wide-blue.png" width="600px">
+</div>
+
+## Introducing Anomalib
+In early 2022, a group of Intel AI Researchers released a ground-breaking Deep Learning Library named Anomalib. The library provides state-of-the art algorithms for Anomaly Detection on Image datasets. Anomalib serves to be a thorough end-to-end solution, offering many features to achieve the highest accuracy model while also providing inference deployment code with Intel’s OpenVino Toolkit.
```
Comment (djdameln) [inline docs/blog/001-train-custom-dataset/README.md:7]: Maybe we should be a bit more humble here. Ground-breaking may be a bit of an overstatement.

File: docs/blog/001-train-custom-dataset/README.md [hunk 5]
```diff
@@ -0,0 +1,245 @@
...
+
+```bash
+python tools/benchmarking/benchmark.py --config tools/benchmarking/benchmark_params.yaml
+```
+
+A nice feature about the provided benchmarking script is that if the host system has multiple GPUs, the runs are parallelized over all the available GPUs for faster collection of result.
+
+**Call to Action**
+
+The Anomalib repository is actively maintained by some of Intel’s top researchers. Their goal is to provide the AI-community with top of the class performance and accuracy while also providing a great user-experience for developers. Checkout out the repo and install anomalib today!
```
Comment (djdameln) [inline docs/blog/001-train-custom-dataset/README.md:245]: Same here. "top researchers" may come across a bit overly confident if we are the ones writing this about ourselves (I mean, it's true of course, but we don't want to be arrogant).

File: docs/blog/001-train-custom-dataset/README.md (hunk 4 above)
Comment (samet-akcay) [reply docs/blog/001-train-custom-dataset/README.md]: Addressed

File: docs/blog/001-train-custom-dataset/README.md (hunk 5 above)
Comment (samet-akcay) [reply docs/blog/001-train-custom-dataset/README.md]: Addressed

**Review by djdameln — APPROVED** [review]

**Pull request merged**